compiler/plugins/target/AMD-AIE/iree-amd-aie/IR/AMDAIEOps.cpp

```diff
@@ -43,6 +43,18 @@ LogicalResult ControlCodeOp::verify() {
 // AMDAIE_CoreOp
 //===----------------------------------------------------------------------===//
 
+// Hardcoded row_offset == 2 -> AIE core rows start from 2
+// TODO(jornt): avoid hardcoding here. Add a device model/identifier to loop up
+// core offset. This will be handled in a follow-up.
```

Comment:
Any doc comment should be in triple quotes. `//` -> `///`

Reply:
Thanks, done.